README.md

```diff
@@ -224,6 +224,12 @@ which are to be added, in that particular order.
 optional_otions is just additional parameters, in object form, which you might
 want to provide to the modules. It's an optional parameter.
 
+Use string syntax (see below) to add multiple steps and configurations quickly by passing a string to `sequencer.addStep()`:
+
+```js
+sequencer.addSteps('invert, channel');
```

Comment:
in theory we should not include spaces here, though maybe it's not such a big deal if it can handle it?

Reply:
Yes, `parseSyntax()` already handles spaces :D